index.html

```diff
@@ -58,155 +74,175 @@ <h2 class="sub-title">I am a Softwaer Developer</h2>
+      
+        <article class="card2  professional-card-5">
+            <div class="card-title">Profesional Art<br> Printing Data</div>
+            <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+            <div class="card-pro-buttons">
+                <ul class="pro-buttons">
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">bootstrap</a></li>
+                    <li><a href="#">Ruby</a></li>
+                    
+                </ul>
+            </div>
+            <a href="#" class="pro-card-button pro-card-desktop">See Project</a>
+        
+           </article>
     
-       </div>
-  </section>
+         <!--professional part start 6-->
+        
+            <article class="card2  professional-card-6">
+                <div class="card-title">Profesional Art<br> Printing Data</div>
+                <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+                <div class="card-pro-buttons">
+                    <ul class="pro-buttons">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">bootstrap</a></li>
+                        <li><a href="#">Ruby</a></li>
+                        
+                    </ul>
+                </div>
+                <a href="#" class="pro-card-button">See Project</a>
+            
+               </article>
+          </div>
+</section>
  
    <!--about section start-->
    <section class="card-3">
-    <div class="card-about-title"><h3>About me</h3></div>
-    <div class="card-about-text"><P>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my works and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</P></div>
-    <div class="work-button">
-        <a href="#" class="card-about-button">Get My Resume</a>
-    </div>
-    <div class="about-image"></div>
-    <div class="about-vector-img"></div>
+          <div class="about-row">
+            <div class="about-frist-row">
+                <div class="card-about-title"><h3>About me</h3></div>
+                <div class="card-about-text"><P>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my works and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</P></div>
+                <div class="work-button">
+                    <button href="#" class="card-about-button">Get My Resume</button>
+                </div>
+                
+            </div>
+            <div class="about-desk-img">
+                <div class="about-vector-img"></div>
+                
+            </div>
+            <div class="about-lower-img"></div>
+          </div>
```

Comment:
- [ ] Kindly add the background images using CSS and not HTML elements since these images are part of the design and they don't contribute to the content of the page 👍 
- [ ] Kindly try to match the design in Figma, the background image position and the width of the section don't match the template 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/215283455-ff2b1d97-576f-4c43-bfd6-51cc9ab4a347.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/215283469-dde4cbde-f08a-4f9d-9ada-4ed13a1e2597.png)